lib/cocoapods/installer/user_project_integrator/target_integrator/xcconfig_integrator.rb

```diff
@@ -29,37 +28,6 @@ def self.integrate(pod_bundle, targets)
           # @!group Integration steps
           #-------------------------------------------------------------------#
 
-          # Removes the xcconfig used up to CocoaPods 0.33 from the project and
-          # deletes the file if it exists.
-          #
-          # @param  [Target::AggregateTarget] pod_bundle
-          #         The Pods bundle.
-          #
-          # @param  [Array<XcodeProj::PBXNativeTarget>] targets
-          #         The native targets.
-          #
-          # @todo   This can be removed for CocoaPods 1.0
```

Comment:
Given 0.39 is millions of downloads and 1.2.0 is 500K downloads I think this is safe.